packaging/hudi-hive-sync-bundle/pom.xml

```diff
@@ -92,6 +92,7 @@
                   <include>com.esotericsoftware:kryo-shaded</include>
                   <include>com.esotericsoftware:minlog</include>
                   <include>org.objenesis:objenesis</include>
+		              <include>com.github.ben-manes.caffeine:caffeine</include>
```

Comment:
what is the reason of this fix? this has to be separate out as another pr to highlight what it fixes. this affects release artifacts